evap/grades/migrations/0009_translate_description.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.1 on 2016-01-04 18:32
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('grades', '0008_translate_description'),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name='gradedocument',
+            unique_together=set([('course', 'description_de'), ('course', 'description_en')]),
+        ),
+        migrations.RemoveField(
+            model_name='gradedocument',
+            name='description',
+        ),
+    ]
```

Comment:
yes, i think that should work. before changing the unique_together we still need the data migration that puts stuff into description_en, but then it should be possible.